README.md

```diff
@@ -27,7 +27,16 @@ This repository contains all of the code that runs on [worldcubeassociation.org]
   ```
   If some changes are made to this hook, you will have to update it running this command from the repository's root directory: `BUNDLE_GEMFILE=WcaOnRails/Gemfile bundle exec overcommit --sign`.
 
-## Run directly with Ruby (lightweight, but only runs the Rails portions of the site)
+## Run using Docker (simplest option, but only runs the Rails portions of the site)
+
+- Install [Docker](https://docs.docker.com/get-docker/) (remember to complete the [Linux post-install steps](https://docs.docker.com/engine/install/linux-postinstall/) if you're on Linux)
+- Install docker-compose. The best way to get an up-to-date version is to use `pip install docker-compose` with Python 3 installed. On some systems, you may need to type `pip3` instead of `pip`.
```

Comment:
You can actually just download the `docker-compose` executable and change permissions as described [here](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-compose-on-ubuntu-20-04#step-1-%E2%80%94-installing-docker-compose). Some Linux distributions come with Python installed, in which case it's equally straightforward, but this sounds the simplest to me. No strong opinion here, so whatever you think is best =)

Reply:
That's actually much better, thank you!